File: io/json/src/test/resources/compat/issue-754/src/java/LegacyJsonCaller.java
package io.bluetape4k.json.compat.issue754.java;

import io.bluetape4k.json.JsonSerializer;

public final class LegacyJsonCaller {
    private LegacyJsonCaller() {
    }

    public static Object deserializeNull(JsonSerializer serializer) {
        return serializer.deserialize(null, Object.class);
    }
}

File: io/json/src/test/resources/compat/issue-754/src/kotlin/LegacyJsonCaller.kt
package io.bluetape4k.json.compat.issue754.kotlin

import io.bluetape4k.json.JsonSerializer
import io.bluetape4k.json.deserialize

fun restoreLegacyJson(serializer: JsonSerializer, payload: ByteArray): String? =
    serializer.deserialize(payload)
